src/lib/replication/repl_dev/raft_repl_dev.h

```diff
@@ -199,6 +212,10 @@ class RaftReplDev : public ReplDev,
     static std::atomic< uint64_t > s_next_group_ordinal;
     bool m_log_store_replay_done{false};
 
+    // pending create requests, including both raft and data channel
+    std::atomic_uint64_t m_pending_init_req_num;
+    std::atomic< bool > m_in_emergency;
```

Comment:
I think we should replace emergency with something else. HomeStore shouldn't have idea about upper layer logic.